src/Sulu/Component/Content/Compat/Structure.php

```diff
@@ -763,17 +763,11 @@ public function setInternal($internal)
      */
     public function getNodeName()
     {
-        if (
-            self::NODE_TYPE_INTERNAL_LINK === $this->getNodeType() &&
-            null !== $this->getInternalLinkContent() &&
-            $this->getInternalLinkContent()->hasProperty('title')
-        ) {
-            return $this->internalLinkContent->getPropertyValue('title');
-        } elseif ($this->hasProperty('title')) {
```

Comment:
Are you sure that this code can be deleted?

Reply:
call undefined method `getInternalLinkContent` so the if statement is never true. or would throw an error?

Reply:
strange is that I also can't find any classes extending this abstract Structure class 🤔. maybe we can remove the whole class?

Reply:
But wouldn't that mean, that an error should be thrown? I don't see how this should then work at all. Are you sure that method is not implemented somehow deep in the inheritance tree, since this is an abstract class?